pages/_app.js

```diff
@@ -1,7 +1,17 @@
+import { ApolloProvider, ApolloClient, InMemoryCache } from '@apollo/client'
 import '../styles/globals.css'
 
+const client = new ApolloClient({
```

Comment:
wanna throw the client + queries into an `graphql` namespace while you're in there?

Reply:
`/graphql/client.js` and `/graphql/queries/transactions.js`?